tests/register.spec.coffee

```diff
@@ -29,7 +29,7 @@ describe 'Device Register:', ->
 
 		before ->
 			mockServer.start().then ->
-				mockServer.post("/device/register?apikey=#{PROVISIONING_KEY}").thenCallback (req) ->
+				mockServer.post('/device/register').thenCallback (req) ->
```

Comment:
Let's just add a check that the authorization header indeed has the correct content and this looks great :+1: